backend/blender_script/main.py

```diff
@@ -9,6 +10,28 @@
 sys.path.append(module_dir)
 from lens import Lens, LensPrescription, Prescription
 
+
+
+#densities are g per cubic meter
+# some densities taken from https://en.wikipedia.org/wiki/Corrective_lens
+IOR_Densities = {'Standard Plastic': 1390000, 
+                 'Polycarbonate': 1200000, 
+                 'High-index Plastic': 1270000, 
```

Comment:
```
'High-index Plastic': 1270000, 
'High-index Plastic': 1370000, 
'High-index Plastic': 1470000,
```
Are these meant to have the same key?

Reply:
Are we indexing with the `Standard Plastic - 1.5` including the `- IOR` or just the material name?

Reply:
Oh yea I wanted to discuss this. I'm thinking it's better if we name each High-index plastic something unique e.g. High-index plastic 1, High-index plastic 2 or something like that. We could index with just the IOR but technically multiple different materials can have the same IOR. We could also index with `Standard Plastic - 1.5` but atm I think the material and IOR are sent separately and I'm too lazy to join them before indexing. lmk which of these u think is good and I can change it to that. Atm `'High-index Plastic': 1470000` overwrites all other previous entries.

Reply:
done, ugly but works